public_suffix_list.dat

```diff
@@ -4150,6 +4153,7 @@ in.na
 cc.na
 tv.na
 ws.na
+mobi.na
 co.na
 com.na
 org.na
```

Comment:
ADD (where ORG.NA is, for sorting) 
gov.na
net.na
org.na

Reply:
actually @groundcat the whole .na section, after confirming it with the administrators of the .NA is as following:

I followed from the IANA.ORG to the .NA listing, to its link to registration services and have placed that in the comment line:

```
// http://www.na-nic.com.na/
na
alt.na
co.na
com.na
gov.na
net.na
org.na
```

Reply:
Thank you @dnsguru ! I have created the PR https://github.com/publicsuffix/list/pull/2204

Reply:
> actually @groundcat the whole .na section, after confirming it with the administrators of the .NA is as following:
> 
> I followed from the IANA.ORG to the .NA listing, to its link to registration services and have placed that in the comment line:
> 
> ```
> // http://www.na-nic.com.na/
> na
> alt.na
> co.na
> com.na
> gov.na
> net.na
> org.na
> ```

What is the "registration services" step? On http://www.na-nic.com.na/ I only see a few links to PDFs but none of them seem to contain this list.

Reply:
Okay, found the relevant comment here: https://github.com/publicsuffix/list/issues/2199#issuecomment-2396379694